lib/api/eval-response.js

```diff
@@ -0,0 +1,30 @@
+import URI from 'urijs';
+
+// make these available for eval
+import _faker from 'faker/build/build/faker';
+import _state from 'api/local-state';
+
+export function evalResponse(responseRule, request) {
+    // Setup local variables for eval
+  let url = URI(request.url);
+  let path = url.segmentCoded();
+  let query = url.search(true);
+  let data = JSON.parse(request.body);
```

Comment:
This will fail on body of:
`data=%7B%22H%22%3A%22tradencyhub%22%2C%22M%22%3A%22login%22%2C%22A%22%3A%5B%22RjlFOTZCOUUtNDVCNC00NUZFLTg5ODMtNTdBMjBEMUY3QzIw%22%5D%2C%22I%22%3A0%7D
Name`